ext/semian/resource.c

```diff
@@ -234,6 +238,25 @@ check_tickets_arg(VALUE tickets)
   return c_tickets;
 }
 
+static int
+check_min_tickets_arg(VALUE min_tickets)
```

Comment:
I don't think we should have a default minimum, I think that should be manually specified by the user.

I think that for our cases, a minimum of something like 5-10 is probably good, but it needs to be examined on a per-resource basis.